lib/libutee/include/tee_api_defines.h

```diff
@@ -135,6 +135,7 @@
 #define TEE_ALG_DES3_CBC_NOPAD                  0x10000113
 #define TEE_ALG_DES3_CBC_MAC_NOPAD              0x30000113
 #define TEE_ALG_DES3_CBC_MAC_PKCS5              0x30000513
+#define TEE_ALG_DES3_CMAC                       0x30000613
```

Comment:
Since this is not defined by the GP spec, I think this should be defined in `tee_api_extensions.h` with a value in the implementation-specific range (updating `__tee_alg_get_class()`, `__tee_alg_get_main_alg()` etc. if needed).

Reply:
Yep, I guess I probably should have discussed this before in Issues.
I also think that it is not the proper place for this define due to the reason you've just mentioned.

Reply:
> Yep, I guess I probably should have discussed this before in Issues.

That's fine, it is often easier to discuss an actual patch ("show me the code!" :smiley:)